refinery/ui/source/js/dashboard/partials/permission-dialog.html

```diff
@@ -46,6 +47,12 @@ <h4>
                   value="none"
                   ng-model="group.permission">
               </td>
+              <td>
+                <input
+                  type="radio"
+                  value="read_meta"
+                  ng-model="group.permission">
+              </td>
```

Comment:
Do you want to add something in `modal-description` up above to describe the new option?

Reply:
Will work on text in the next pull request which is focused on the UI. Same modal will be used in the file browser also.